src/components/dialog-full-screen/content.style.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+const StyledContent = styled.div`
+  padding: 30px 0;
+  overflow-y: auto;
+  height: calc(100% - 60px);
+
+  .carbon-full-screen-heading + & {
```

Comment:
Does this work? It looks like you've removed this class [here](#diff-1e7a40a8b851bfedb09792299b8253eb).

Reply:
I removed it. Thanks,

Reply:
Good point, thank you. I've fixed it.